feat: support search location

Tubt opened the pull request.

Tubt committed:
feat: support search location

ghost commented:
**⏩ No tests generated (cf3f72cd7a2f2879e1bd84fe346afce1441a928b) [View output ↗](https://app.usetusk.ai/app/testing-commit-check/d6f8ce66-72a6-4881-8420-f9ba60d48b39?client=9580ed98-2e58-440b-9abe-374e4bc31365)**

> [!TIP]
> New to Tusk? Learn more [here](https://docs.usetusk.ai/automated-tests/overview).

---

<details><summary>Existing test issues</summary>

Tests generated in these files will include these fixes. If you wish to only commit these fixes, you can do so in the Tusk app.

Tests for these symbols were not generated because the existing test files had errors and were failed to fix:

- `test/e2e/integration/homepage.cy.ts` - (`HomePage`)
- `test/e2e/integration/search.cy.ts` - (`SearchSection`)

The issues in these test files must be resolved before Tusk can generate tests for these symbols.

Please contact [Support](https://app.usetusk.ai/app/testing-commit-check/d6f8ce66-72a6-4881-8420-f9ba60d48b39?client=9580ed98-2e58-440b-9abe-374e4bc31365&showChatBubble=true) if you have questions.

</details>

<details><summary>View check history</summary>
<p>

| Commit | Status | Output | Created (UTC) |
|--------|--------|--------|--------|
| 9c9dc414dcb03553af69fdeb42306b1e2e1b5447 | ⏩ Test environment setup failed | [Output](https://app.usetusk.ai/app/testing-commit-check/35eaa2a0-e0ce-460d-b8fb-ba2c9e248c36?client=9580ed98-2e58-440b-9abe-374e4bc31365) | Oct 16, 2025 3:33AM |
| ae2c7ee7ded87297edf3bdf049fd924dee50f39b | ⏩ Test environment setup failed | [Output](https://app.usetusk.ai/app/testing-commit-check/7baf05c9-8ee2-4301-be00-ade357a5f316?client=9580ed98-2e58-440b-9abe-374e4bc31365) | Oct 16, 2025 4:03AM |
| 12830a5afe3d5034b691572a5d198d185756d349 | ⏩ Test environment setup failed | [Output](https://app.usetusk.ai/app/testing-commit-check/8f7fb50f-7f0a-4241-952f-d6c32cb25783?client=9580ed98-2e58-440b-9abe-374e4bc31365) | Oct 16, 2025 4:22AM |
| a6b1692f6bb08d40197cee00a15609e1f7cc00ba | ⏩ No tests generated | [Output](https://app.usetusk.ai/app/testing-commit-check/ac3f4cb8-1c00-40b9-ac88-cd8d440562e1?client=9580ed98-2e58-440b-9abe-374e4bc31365) | Oct 16, 2025 4:33AM |
| cf3f72cd7a2f2879e1bd84fe346afce1441a928b | ⏩ No tests generated | [Output](https://app.usetusk.ai/app/testing-commit-check/d6f8ce66-72a6-4881-8420-f9ba60d48b39?client=9580ed98-2e58-440b-9abe-374e4bc31365) | Oct 16, 2025 5:00AM |

</p>
</details>

[View output in GitHub ↗](https://github.com/Tubt/travel-app/pull/16/checks?check_run_id=52877768442)

Tubt committed:
tusk config vite

- Added search functionality to SearchSection component
- Search filters destinations by name and description
- Added data-testid attributes for testing

@tusk Generate Cypress E2E test for the search functionality that:
- Tests searching by destination name
- Tests searching with Vietnamese characters
- Tests no results scenario
- Tests clearing search
- Tests Enter key and button click
